package.json

```diff
@@ -10,15 +10,15 @@
     "build:compare": "node script/content-build-compare.js && diff --color websiteContentBuildHash.txt standaloneContentBuildHash.txt",
     "build:content": "node --max-old-space-size=8192 script/build-content.js",
     "build:content:test": "node script/test-cms-export.js",
-    "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js",
-    "build:webpack:prod": "NODE_ENV=production yarn build:webpack --env.buildtype vagovprod",
-    "build:webpack:staging": "yarn build:webpack --env.buildtype vagovstaging",
-    "build:webpack:dev": "yarn build:webpack --env.buildtype vagovdev",
-    "build:webpack:local": "yarn build:webpack --env.buildtype localhost",
+    "build:webpack": "NODE_OPTIONS=--max-old-space-size=8192 webpack --config config/webpack.config.js",
```

Comment:
Why the increase from `4096` to `8192` only here?

Reply:
I agree that we should look at the memory usage. I'm not sure what's causing us to use so much but it really seems like multiple gigs shouldn't be required for this build process.

Reply:
> Oof. Moar memory!!!
> 
> Out of scope for this PR, but we really need to figure out how to reduce the memory footprint of the webpack build. It's only going to get worse and worse as we add more applications to the project.

I thought we were planning to split up the apps into separate builds?

Reply:
Our current effort is to separate the content build from the app build. I know there's also been some discussion about potentially splitting the apps into separate repos but I don't know all the background on that. Webpack 5 provides [module federation](https://webpack.js.org/concepts/module-federation/) which is one possible build process for such a configuration.